_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each claim for reimbursement from congresspeople and of identifying it’s illegal probability.
+  last_histories: Last stories
+  loading: Loading…
+  more_posts: More stories
```

Comment:
Here I don't mind: technically and semantically _posts_ and _stories_ are valid names.